lib/atscppapi/examples/async_http_fetch/AsyncHttpFetch.cc

```diff
@@ -214,6 +214,7 @@ void
 TSPluginInit(int argc ATSCPPAPI_UNUSED, const char *argv[] ATSCPPAPI_UNUSED)
 {
   TS_DEBUG(TAG, "Loaded async_http_fetch_example plugin");
+  atscppapi::TSRegisterPlugin( std::string("CPP_Example_AsyncHttpFetch"),  std::string("apache"),  std::string("info@example.com"));
```

Comment:
#. There's a `using namespace atscppapi` at the head of the file so you don't need to qualify this call.
#. Literal strings will auto promote to `std::string` arguments so no need to do that explicitly.
#. Make the email address "users@trafficserver.apache.org"
#. If you can, clang format the files as you edit.